server/data/data-jmap/src/main/java/org/apache/james/jmap/api/vacation/Vacation.java

```diff
@@ -100,9 +100,6 @@ public Builder copy(Vacation vacation) {
 
         public Vacation build() {
             boolean enabled = isEnabled.orElse(DEFAULT_DISABLED);
-            if (enabled) {
```

Comment:
I am sorry that it do not appear here. It had. In unit tests and integration tests.

I guess it is a rebase failut, and I committed tests modifications with other diff.

Do you want me to move it explicitly to this commit ?